crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -769,31 +783,33 @@ mod tests {
 		let temp_dir = TempDir::new()?;
 		let temp_dir_path = temp_dir.path();
 
-		// No directory path.
-		let file = temp_dir_path.join("chain-spec.json");
-		let result = prepare_output_path(&file)?;
-		// Expected path: chain-spec.json
-		assert_eq!(result, file);
-
-		// Existing directory Path.
-		for dir in ["existing_dir", "existing_dir/", "existing_dir_json"] {
-			let existing_dir = temp_dir_path.join(dir);
-			create_dir_all(&existing_dir)?;
-			let result = prepare_output_path(&existing_dir)?;
-			// Expected path: existing_dir/chain-spec.json
-			let expected_path = existing_dir.join(DEFAULT_SPEC_NAME);
-			assert_eq!(result, expected_path);
-		}
-
-		// Non-existing directory Path.
-		for dir in ["non_existing_dir", "non_existing_dir/", "non_existing_dir_json"] {
-			let non_existing_dir = temp_dir_path.join(dir);
-			let result = prepare_output_path(&non_existing_dir)?;
-			// Expected path: non_existing_dir/chain-spec.json
-			let expected_path = non_existing_dir.join(DEFAULT_SPEC_NAME);
+		let test_cases = [
```

Comment:
Might be a good idea to throw in a more complex path, like `("existing_dir/subdir/", "existing_dir/subdir/chain-spec.json")`. I tried it out and it works, but it’d add a bit more coverage.

Reply:
Included in https://github.com/r0gue-io/pop-cli/pull/357/commits/b1969c0d820fbc30a1c9ab7ceaa60126307dca53